app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,61 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+
+require foundation_and_overrides
+
+ */
+@import "colors";
+@import url(https://fonts.googleapis.com/css?family=Mako);
+
+a {
+  color: #FFE5AA;
+}
+
+p {
+  display: inline;
+}
+
+.back-1 {
+   background-image: url('http://images.fineartamerica.com/images-medium-large/blue-city-david-lee-thompson.jpg');
+   background-size: cover;
+   height: auto !important;
+   margin-left: 0px;
+   margin-right: 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>`0px` should be written without units as `0`